index.js

```diff
@@ -58,22 +58,18 @@ getNewDoggy()
 
 likeBtn.addEventListener("click", () => { 
     doggy.hasBeenLiked = true
-    if (doggy.hasBeenLiked){
-        doggy.likeRender()
-        setTimeout(() =>{
-            getNewDoggy()
-        },2000)
+    if (doggy.hasBeenLiked) {
```

Comment:
just the suggestion not the must have

if you have one line action inside you can avoid {} and indents.